### DISCUSSION TITLE
1.2.1升级到1.2.7.1版本时，FE启动报错

### DISCUSSION THREAD

**Discussion by w875050772** [apache/doris#26834] [A - General / Q&A]
启动报错内容如下：
Error option /bin/bash /opt/apache-doris/fe/bin/start_fe.sh
DEBUG >>>>>> FE_ID = [0]
DEBUG >>>>>> FE_SERVERS = [fe1:xxx.xxx.xxx.xx:9010]
DEBUG >>>>>> tmpFeId = [1]
DEBUG >>>>>> tmpFeIp = [xxx.xxx.xxx.xx]
DEBUG >>>>>> tmpFeEditLogPort = [9010]
DEBUG >>>>>> feIpArray = xxx.xxx.xxx.xx
DEBUG >>>>>> feEditLogPortArray = 9010
DEBUG >>>>>> masterFe = xxx.xxx.xxx.xx:9010
DEBUG >>>>>> currentFe = :
DEBUG >>>>>> Append the configuration [priority_networks = ...0/24] to /opt/doris-fe/conf/fe.conf
DEBUG >>>>>> FE is follower, fe_id = 0
DEBUG >>>>>> registerMySQL = 【mysql -uroot -P9030 -p123456 -hxxx.xxx.xxx.xx -e "alter system add follower ':'"】
DEBUG >>>>>> registerShell = 【/opt/apache-doris/fe/bin/start_fe.sh --helper 'xxx.xxx.xxx.xx:9010'】
DEBUG >>>>>> feMasterStat =  【mysql -uroot -P9030 -p123456 -h xxx.xxx.xxx.xx -e "show frontends" | grep "xxx.xxx.xxx.xx_9010" | grep -E "true[[:space:]]*true"】
Run registerShell command, [ registerMySQL = mysql -uuser -P9030 -p123456 -hxxx.xxx.xxx.xx -e "alter system add follower ':'" ]
ERROR 2002 (HY000): Can't connect to MySQL server on 'xxx.xxx.xxx.xx' (115)
ERROR 2002 (HY000): Can't connect to MySQL server on 'xxx.xxx.xxx.xx' (115)
Run registerShell command, [ registerMySQL = mysql -uroot -P9030 -p123456-hxxx.xxx.xxx.xx -e "alter system add follower ':'" ]
ERROR 2002 (HY000): Can't connect to MySQL server on 'xxx.xxx.xxx.xx' (115)
ERROR 2002 (HY000): Can't connect to MySQL server on 'xxx.xxx.xxx.xx' (115)

**Comment by ixzc** [comment]
add my wechat: Faitch_xzc
